scripts/installscripts/stretch-install-default.sh

```diff
@@ -440,6 +440,20 @@ sudo cp /home/pi/RPi-Jukebox-RFID/misc/sampleconfigs/php.ini.stretch-default.sam
 sudo chown root:root /etc/php/7.0/fpm/php.ini
 sudo chmod 644 /etc/php/7.0/fpm/php.ini
 
+# settings for opcache
+# we cache to memory and disk (level 2) for faster upstart
+sudo mkdir /home/pi/RPi-Jukebox-RFID/.php-opcache/
+sudo chmod 1777 /home/pi/RPi-Jukebox-RFID/.php-opcache/
+sudo touch /etc/php/7.0/fpm/conf.d/999-RPi-Jukebox-RFID.ini
```

Comment:
fpm was removed now in #839, should this be removed also here?

Reply:
though these configs would be better of in the `/etc/php/7.0/cgi` directory